code/game/machinery/computer/records.dm

```diff
@@ -44,7 +44,7 @@
 
 	icon_screen = "security"
 	light_color = LIGHT_COLOR_ORANGE
-	req_one_access = list(access_security, access_forensics_lockers, access_lawyer, access_hop)
+	req_one_access = list(access_security, access_weapons, access_forensics_lockers, access_lawyer, access_hop)
```

Comment:
Dont use `access_weapons` for that.
`access_weapons` is used to determine if someone is allowed to use weapons, not if they are a security officer.